engine/classes/Elgg/Notifications/Event.php

```diff
@@ -1,12 +1,14 @@
 <?php
 namespace Elgg\Notifications;
+
 /**
- * Notification event
+ * Subscription notification event
  * 
  * @package    Elgg.Core
  * @subpackage Notifications
  */
-class Event {
+class Event implements NotificationEvent {
```

Comment:
If the goal is to keep this `Event` class for BC, just make a `SubscriptionNotificationEvent` that simply extends `Event` and use it instead.